source/_components/media_player.toshiba.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: "Toshiba Cast TV"
+description: "Instructions on how to integrate a Toshiba Cast TV into Home Assistant."
+date: 2017-08-13 13:04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Media Player
+featured: false
+ha_release: 0.17
+ha_iot_class: "Local Polling"
+---
+
+The `toshiba` platform allows you to control a Toshiba Cast TV. The component allow to query and set the power, the input source and the volume.
```

Comment:
> The component allow to query

`allow` => `allows`, or you could change the wording, since `allows` is repetitive already.
You could replace it with `The component can query`

---

> the power, the input source and the volume.

This sentence contains a series of multiple words; therefore a comma needs to be placed before the `and`.
So it becomes:

> the power, the input source, and the volume.